geomesa-quickstart-kafka/pom.xml

```diff
@@ -75,7 +76,7 @@
                             <arguments>
                                 <argument>-classpath</argument>
                                 <classpath />
-                                <argument>com.example.geomesa.kafka.KafkaQuickStart</argument>
+                                <argument>org.geomesa.example.kafka.KafkaQuickStart</argument>
                                 <argument>-kafka.brokers</argument>
                                 <argument>${brokers}</argument>
```

Comment:
Also maybe `$[kafka.brokers}` and `${kafka.zookeepers}`?

Reply:
good catch, thanks